cpp/ps0000/ps0000_solv.cpp

```diff
@@ -15,34 +16,38 @@ using namespace std;     // Missing
 */
 
 
-int main(void)          // incorrect
+// int Main(void)          // incorrect
 
-int main()             // correct
+int main(void)             // correct
 
 /* First error  : If you used void inside main function you must change int --> to be --> void too.
    Second error : It tells you that no value is returned for this program so the command return must be deleted .
-   Third error  : In this program you need to return a value so must delete void & put int .
+
+   1/1
 */
 
 {
-    cout >> "Hello , world! /n"   // incorrect
+    // cout >> "Hello , world! /n"   // incorrect
 
-    cout << "Hello , world! \n"; // correct
+    cout << "Hello, World! \n"; // correct
 
     /* First error  : The brackets used here are for the cin/input command, not the cout/output command.
        Second error : You are writing the wrong slant direction, it should be from this / to this \ .
        Third error  : You forgot the semicolon that is used to specify that this is the end of the requested command
                       and so that it moves to the next command.
+       3/4
    */
 
 
-    return 1;     // incorrect
+    // return 1;     // incorrect
 
     return 0;    //  correct
 
     /* First  : You are writing the wrong value because return 0 in the main function means that the program executed successfully
                 and return 1 in the main function means that the program does not execute successfully and there is some error.
        Second : You can use return 1 in a user-defined function because returning 1 means that the user-defined function returns true
                 and returning 0 means that the user-defined function returns false in the case of user-defined function too .
+       1/1
     */
 }
+// great work 😎 10/10 ⭐

```

Comment:
- [x] most of code errors and bugs are fixed
- [x] file is in the right path
- [x] +1 mark for your first commit
+1 bonus mark for your comments 😊